### PR TITLE
feat(14751): Completions on class: put statics first

### LINKED ISSUE #14751: Completions on class: put statics first
**TypeScript Version:**  nightly (typescript@2.3.0-dev.20170320)

**Code**

```ts
class C {
  static foo() {}
  static bar() {}
}

C.
```

**Expected behavior:**

Completions `foo` and `bar` are first.

**Actual behavior:**

They are hidden among `apply`, `arguments`, `bind`, `call`, `caller`, `length`, `name`, `prototype`, and `toString`.

### PR DESCRIPTION
**Pull request by a-tarasyuk**
Fixes #14751

### CODE REVIEW

File: src/services/completions.ts [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
-        AutoImportSuggestions = "5",
-        JavascriptIdentifiers = "6"
+        LocalDeclarationPriority = "0",
+        LocationPriority = "1",
+        OptionalMember = "2",
+        MemberDeclaredBySpreadAssignment = "3",
+        SuggestedClassMembers = "4",
+        GlobalsOrKeywords = "5",
+        AutoImportSuggestions = "6",
+        JavascriptIdentifiers = "7"
```
Comment (andrewbranch) [inline src/services/completions.ts:11]: @uniqueiniquity is this change something that would impact IntelliCode/VS tooling?

Comment (uniqueiniquity) [reply src/services/completions.ts:11]: Yep 😫 let me see if I can characterize what will need to change and I'll circle back.

Comment (uniqueiniquity) [reply src/services/completions.ts:11]: Good to go on the VS side for this change 👍

**Review by andrewbranch — APPROVED** [review]

File: src/services/completions.ts (hunk 1 above)
Comment (andrewbranch) [reply src/services/completions.ts:11]: Thanks!

**Pull request merged**